packages/tx/README.md

```diff
@@ -207,6 +208,47 @@ const tx = FeeMarketEIP1559Transaction.fromTxData(txData, { common })
 console.log(bytesToHex(tx.hash())) // 0x6f9ef69ccb1de1aea64e511efd6542541008ced321887937c95b03779358ec8a
 ```
 
+#### EOA Code Transaction (EIP-7702) (experimental)
+
+- Class: `EOACodeEIP7702Transaction`
+- Activation: `prague` (or per EIP setting)
+- Type: `4`
+
+This library suppports an experimental version of [EIP-7702](https://eips.ethereum.org/EIPS/eip-7702) starting with `v5.4.0`. This tx type allows to run code in the context of an EOA and therefore extend the functionality which can be "reached" from respectively integrated into the scope of an otherwise limited EOA account.
```

Comment:
For experimental versions of EIPs, I suggest we point to a permalink on GitHub such that we point to the spec which the EIP is actually implemented in. (Current spec is very different)

Reply:
I have now kept both (I think the permalink to the respective EIP is also interesting for people as a reference), differentation should be though from the text.